docs/src/views/Colors.tsx

```diff
@@ -0,0 +1,352 @@
+              </li>
+              <li>
+                {code('foreground')} is a{' '}
+                <b>
+                  color token (theme-dependent value) that should be used as value in styles. Note
+                  that actual token's value is dependent on theme.
+                </b>{' '}
+                You can see an example of tokens defined for Teams theme{' '}
+                <Text as={Link} weight="bold" content="here" color="brand" to="color-schemes" />.
+              </li>
+            </ul>
+          </p>
+          <p>
+            The color tokens defined in the color scheme are mapped to actual values for all themes
+            used in the application. This means that, if the developers use some token from the
+            schema, it will be mapped to the correct color value provided by the current theme.
+          </p>
+
+          <p>Let's illustrate this with code example:</p>
+
+          <ExampleSnippet
+            value={`
+            import React from 'react'
+            import { Button } from '@stardust-ui/react'
+
+            const theme = {
+              componentStyles: {
+                Box: {
+                  root: ({ theme: { siteVariables } }) => {
+                    const colorScheme = siteVariables.colorScheme
+                    return {
+                      color: colorScheme.brand.foreground3,
+                      backgroundColor: colorScheme.brand.background,
+                      border: \`1px solid \${colorScheme.brand.border}\`,
+                      ':hover': {
+                        color: colorScheme.brand.foregroundHover,
+                        backgroundColor: colorScheme.brand.backgroundHover,
+                        border: \`1px solid \${colorScheme.brand.borderHover}\`,
+                      },
+                    }
+                  },
+                },
+              },
+            }
+
+            const ColorSchemeExample = () => (
+              <Provider theme={mergeThemes(themes.teams, theme)}>
+                 <Text content={'Box in light theme'} />
+                 <Box content={'LIGHT THEME - HOVER ME'}/>
+                 <Text content={'Box in high contrast theme'} />
+                 <Provider theme={themes.teamsHighContrast}>
+                   <Box content={'HIGH CONTRAST THEME - HOVER ME'}/>
+                 </Provider>
+              </Provider>
+            )
+
+            export default ColorSchemeExample
+        `}
+            render={() => (
+              <Provider theme={mergeThemes(themes.teams, theme)}>
+                <Text content={'Box in light theme'} />
+                <Box content={'LIGHT THEME - HOVER ME'} />
+                <Text content={'Box in high contrast theme'} />
+                <Provider theme={themes.teamsHighContrast}>
+                  <Box content={'HIGH CONTRAST THEME - HOVER ME'} />
+                </Provider>
+              </Provider>
+            )}
+          />
```

Comment:
I don't understand what happens in this example. Do we need `:hover` there? How we can make it really simple?

Reply:
I wanted to show at least two states, hover seems like the most simple for example.

Reply:
We share in this examples too many concepts. 

1. '::hover' selector, is it a part related to color scheme? 
2. 
``` 
     root: ({ theme: { siteVariables } }) => {
        const colorScheme = siteVariables.colorScheme
``` 

We showing in our example direct usage of `componentStyles` (💣 ), nesting is 🙈 

```jsx
componentVariables: (siteVariables) => ({
  Box: {
    color: siteVariables.colorScheme.brand.foreground3,
    backgroundColor: siteVariables.colorScheme.brand.background,
  })
},
componentStyles: ({ variables }) => ({
  color: variables.color,
  backgroundColor: variables.backgroundColor,
})
```

Can we try to utilize real pattern?

Reply:
![image](https://user-images.githubusercontent.com/4512430/59516704-37a6f900-8ec2-11e9-95b5-770c8f303071.png)
Is it better now?

Reply:
![image](https://user-images.githubusercontent.com/4512430/59518439-ebf64e80-8ec5-11e9-8659-8cdaf02bb212.png)
 Final look :)